modules/behavioural_qc/php/provisioners/behaviouralprovisioner.class.inc

```diff
@@ -0,0 +1,84 @@
+<?php declare(strict_types=1);
+/**
+ * PHP version 7
+ *
+ * @category Behavioural
+ * @package  Loris
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+namespace LORIS\behavioural_qc\Provisioners;
+
+/**
+ * The BehaviouralProvisioner class
```

Comment:
That much is obvious.. what does the BehaviouralProvisioner class do? What is it for?